Admin/AbstractAdmin.php

```diff
@@ -1042,7 +1042,7 @@ public function getActiveSubclassCode()
     /**
      * {@inheritdoc}
      */
-    public function getBatchActions()
+    final public function getBatchActions()
```

Comment:
@core23 yes I know it's for next major but it's not a reason to not make a BC way on the stable branch before the major. Especially for methods like `getBatchActions`.

This is why I'm asking that.

The goal is the same as Symfony: Make the last minor, if possible, the same as the first major. (e.g. SF 2.8 and 3.0).